dist/index.html

```diff
@@ -68,24 +69,42 @@ <h1>Testing the Nostr Embed</h1>
         const a = document.getElementsByTagName('script')[0];
         a.parentNode.insertBefore(n, a);
       })();
-    </script>
+    </script-->
 
-    <script>
+    <!-- script>
```

Comment:
Yes let's leave them for now, until I figure out how to throttle lots of embeds on a single page so that they wouldn't overload the relay with a storm of requests and wouldn't timeout.